python/pyarrow/table.pxi

```diff
@@ -1331,7 +1331,7 @@ cdef class Table(_PandasConvertible):
 
     def replace_schema_metadata(self, metadata=None):
         """
-        EXPERIMENTAL: Create shallow copy of table by replacing schema
+        Create shallow copy of table by replacing schema
```

Comment:
Minor nit: it looks like it's still marked experimental in table.h, maybe remove that too? 
https://github.com/apache/arrow/blob/32679ddf0495a50b2158146709e7ecfd27a467d9/cpp/src/arrow/table.h#L154-L160

Reply:
Good point (I actually noted it on the JIRA as well, but forgot about it :-)). Will update.